src/target/cortexm.c

```diff
@@ -476,6 +480,7 @@ bool cortexm_probe(ADIv5_AP_t *ap)
 		PROBE(lpc17xx_probe);
 	}
 #undef PROBE
+	target_add_ram(t, 0x00000000, 0); /* Provide some memory map to gdb*/
```

Comment:
This is unguarded - that feels very bad and we can foresee this causing problems